### ISSUE TITLE
Suggestion Box Feedback: Cover Sheet Location

### ISSUE THREAD

**Issue by megmcclell** [BYU-IS-Media/independent_study#146]
This was a suggestion we received: 
"It is very difficult to find the cover sheet to print out, or the online submitting of paper course portfolios. I'm not sure working it under "articles" is the best way to be clear about it!"

I honestly had no idea if anyone was even using those cover sheet links in the first place, but it has come up to me as a supervisor multiple times as well as this suggestion we received since we changed it. 

I do think Articles is kind of unclear as far as that is where people should be looking for additional support questions. Have we considered other titles for those links? I'm not even sure people understand that they are a link to more information. Thoughts?

**Comment by tessakohler1** [comment]
Yes, articles is really generic...do you have any ideas for button text that would work better?

**Comment by megmcclell** [comment]
Luke and I were just discussing, maybe saying:
"More Support Resources"
"More Counselor Resources"
Etc...

Or maybe even a "View All _______ Resources" would be better, but it's longer than more?

Also, Luke thinks the buttons should be actions instead of informational. So under each category, the button would maybe the most functionally used action for each category, something like:

Support>>          Order Transcript
Counselor>>      Check a Student's Progress
Tech Support>> ???
Testing>>           Find a Testing Center
Blog>>               ???

So we're thinking each category has one button, most common action we want them to perform through the support page, and then has the other bullets to click on and the last bullet just says:

View All Support Resources
View All Counselor Resources
etc...

**Comment by jvthomas93** [comment]
@luke80 when a user searches "Cover Sheet" the form doesn't show up. I feel like it would help resolve the difficulty of finding it. 

This is because it is accessed through isreg.byu.edu. Do we know when the form will be switched over?
